CHANGELOG.rst

```diff
@@ -82,7 +82,7 @@ Changelog
 1.1.3 (2018-10-19)
 ==================
 
-* Adapt Bootstrap4Picture to reflect djangocms_picture.AbstractPicture changes regarding Responsive Images (#65)
+* Adapt Bootstrap5Picture to reflect djangocms_picture.AbstractPicture changes regarding Responsive Images (#65)
```

Comment:
I don't think it's necessary to rename Bootstrap4 to Bootstrap5 in a changelog entry from 2018.